cmd/comment/show.go

```diff
@@ -0,0 +1,65 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmdcomment
+
+import (
+	"path/filepath"
+
+	"github.com/elastic/cloud-sdk-go/pkg/api/commentapi"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/ecctl/pkg/ecctl"
+)
+
+var showCmd = &cobra.Command{
+	Use:     "show <id> --resource-type <resource-type> --resource-id <resource-id>",
+	Short:   "Shows a resource comment",
```

Comment:
This is an ECE only command, we need to specify that by using [this](https://github.com/elastic/ecctl/blob/master/cmd/util/help_text.go#L31-L33) function.
```suggestion
	Short:   cmdutil.AdminReqDescription("Shows information about a resource comment"),
```

Reply:
Thanks a lot for the sharp 👀  @karencfv . For my own education, how do you figure out if an API is only ECE specific?

Reply:
if the endpoint is not in https://github.com/elastic/cloud-sdk-go/blob/master/api/apidocs-user.json then the API is ECE only.